components/StyledLinks/index.js

```diff
@@ -0,0 +1,14 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export const StyledLink = styled(Link)`
+  color: black;
+  text-decoration: none;
+`;
+
+export const StyledBackLink = styled(StyledLink)`
+  display: flex;
+  align-items: center;
+  width: 12.5rem;
```

Comment:
That's not a good solution. This is not flexible anymore to have a fixed width of 12.5 rem. Instead better make use of `display: inline-flex`.